aim/storage/treeview.py

```diff
@@ -42,7 +54,7 @@ def __getitem__(
         self,
         path: Union[AimObjectKey, AimObjectPath]
     ) -> AimObject:
-        ...
+        return self.collect(path)
```

Comment:
can we also implement `get()` via `__getitem__`?

Reply:
Thanks, I missed that.